overleaf.el

```diff
@@ -188,6 +188,11 @@ or `overleaf-read-cookies-from-chromium' is used."
   :type 'boolean
   :group 'overleaf-mode)
 
+(defcustom overleaf-webdriver-init-function #'overleaf--webdriver-make-session
+  "Function called to create a webdriver session."
```

Comment:
I'd add a second line that states the function should return a webdriver-session instance.